reference/examples/LazyCreateAccount2.md

```diff
@@ -0,0 +1,14 @@
+Lazy-create a new account using a public-address via the CryptoTransfer transaction.
+
+Reference: HIP-583
+
+## Example 1
+- Create an Ethereum Account Address / public-address (example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Use the `TransferTransaction` and transfer tokens to the Ethereum Account Address / public-address
+- Request the child reciept or child record to get the Hedera Account ID for this account
+- Show the newly created account does not have a pubic key associated with it by getting the account info and returning the public key field
+- Transfer tokens to the public-address for the account that was created using the `TransferTransaction`
+    - Use the newly created account as the transaction fee payer
```

Comment:
What account/key/address is signing the transfer transaction so that the public key can be extracted and applied?